modules/client/src/main/java/org/apache/ignite/internal/client/table/AbstractClientView.java

```diff
@@ -50,6 +63,36 @@ abstract class AbstractClientView<T> implements CriteriaQuerySource<T> {
         this.tbl = tbl;
     }
 
+    /**
+     * Get index mapping from result set to schema.
+     *
+     * @param columns Columns to map.
+     * @param metadata Metadata for query results.
+     * @param startInclusive The first index to cover.
+     * @param endExclusive Index immediately past the last index to cover.
+     * @return Index mapping.
+     */
+    protected static List<Integer> indexMapping(
```

Comment:
it looks as utility method - let's move it form the class

Reply:
Moved index mapping logic into `SqlRowProjection`